release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -1285,6 +1285,33 @@ conditions:
 [id="ocp-4-9-web-console-admin-perspective-bug-fixes"]
 ==== Web console (Administrator perspective)
 
+* Previously, all rows on the `PF4` table were rerendering. With this update, the content in `React.memo` was wrapped so the content does not rerender on every scroll event.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1856355[*BZ#1856355*])
+
+* Previously, the charts in *Cluster Utilization* in the {product-title} web console displayed the data time span in a confusing manner. For example, if the six hour time span option is selected, but data exists only for the final three hours, those three data points are stretched to fill the entire chart. The first three hours are not displayed. This could result in the assumption that the chart is showing the full six-hour time span. To avoid confusion, the charts now show blank space for missing information. In this example, the chart displays the entire six-hour time span with data starting at the fourth hour. The first three hours are blank.
```

Comment:
```suggestion
* Previously, the charts in *Cluster Utilization* in the {product-title} web console displayed the data time span in a confusing manner. For example, if the six-hour time span option is selected, but data exists only for the final three hours, those three data points are stretched to fill the entire chart. The first three hours are not displayed. This could result in the assumption that the chart is showing the full six-hour time span. To avoid confusion, the charts now show blank space for missing information. In this example, the chart displays the entire six-hour time span with data starting at the fourth hour. The first three hours are blank.
```
(Matches consistency with later instances in this paragraph.)